polaris/ocean/convergence/__init__.py

```diff
@@ -82,7 +82,11 @@ def get_timestep_for_task(config, refinement_factor, refinement='both'):
     section = config['convergence_forward']
     time_integrator = section.get('time_integrator')
     # dt is proportional to resolution: default 30 seconds per km
-    if time_integrator == 'RK4':
+    if (
+        time_integrator == 'RK4'
+        or time_integrator == 'LTS'
+        or time_integrator == 'FB_LTS'
+    ):
         dt_per_km = section.getfloat('rk4_dt_per_km')
         btr_timestep = 0.0
```

Comment:
@cbegeman, I made this small change to get things working in advance of figuring out the best way to handle this here. Before now, RK4 was the only time-stepping scheme to not use a barotropic/baroclinic splitting.

I'd propose adding an `elif` for the `LTS` and `FB_LTS` cases that sets `dt_per_km` from something like `lts_dt_per_km` in the local config files? We could also change `rk4_dt_per_km` to `unsplit_dt_per_km`, but I'd imagine that would require changing a lot of cases. Thoughts?

Reply:
Should be taken care of in [10aeec8](https://github.com/E3SM-Project/polaris/pull/325/commits/10aeec8f5ebfad6cd1c3ba9b28120f9ca69a45b1). Let me know if you see anything else!